lib/utils/datatypes.js

```diff
@@ -12,6 +12,7 @@ const isDefined = (data) => typeof data !== "undefined" && data !== null;
 const isUndefined = (data) => typeof data === "undefined";
 const isNumber = (data) => typeof data === "number";
 const isBoolean = (data) => typeof data === "boolean";
+const isFile = (data) => data instanceof File || data instanceof Blob;
 
 const isEncryptable = (data) =>
   isDefined(data) && (isString(data) || isNumber(data) || isBoolean(data));
```

Comment:
Do we need `isFile` in here too? Noticed the `isFile` check in `crypto.js` is inside an `isEncryptable` check

Reply:
I decided to just remove it from the one in `crypto.js` because it would've ended up encrypting Files that are contained within Objects which is a bit of a strange behaviour.